docs/content/en/docs/whatsnew/changelog.md

```diff
@@ -14,6 +14,80 @@ Please upgrade to `v0.15.3` if you are using Tinkerbell (Bare Metal) provider.
 * Installing CSI as part of VSphere cluster creation was deprecated in version `v0.16.0` and has been removed in `v0.17.0`. Please refer to the [deprecation section]({{< relref "../getting-started/vsphere/vsphere-getstarted/#vsphere-csi-driver-deprecation" >}}) in the vSphere provider documentation for more information.
 {{% /alert %}}
 
+## [v0.17.0](https://github.com/aws/eks-anywhere/releases/tag/v0.17.0)
+
+### Supported OS version details
```

Comment:
Lets add a note here somewhere, and please check the upgrade docs for this as well, but basically saying that when upgrading to 0.17, or any version for that matter, a new image must be created using the new image-builder cli.